static/js/trees.js

```diff
@@ -0,0 +1,575 @@
+            }
+        ]
+    }
+}
+
+const JOIN_US_CARDS = [
+    {
+        className: "fa-brands fa-github",
+        content: "Contribute to <br/> GitHub",
+        href: "https://github.com/kairos-io/kairos"
+    },
+    {
+        className: "fa-solid fa-envelope",
+        content: "Sign up to <br/> our newsletter",
+        href: "https://kairoslinux.substack.com/p/karios-chronicles-were-part-of-the?sd=pf"
+    },
+    {
+        className: "fa-brands fa-twitter",
+        content: "Follow us <br/> on Twitter",
+        href: "https://twitter.com/Kairos_OSS"
+    },
+    {
+        className: "fa-brands fa-slack",
+        content: "Join our <br/> Slack channel",
+        href: "https://spectrocloudcommunity.slack.com/join/shared_invite/zt-g8gfzrhf-cKavsGD_myOh30K24pImLA#/shared-invite/email"
+    },
+].map(({className, content, href})=>({
+    a: {
+        href,
+        children: [
+            {
+                i:{
+                    class: className,
+                }
+            },
+            {
+                div: {
+                    children:[
+                        {
+                            p:{
+                                content
+                            }
+                        }
+                    ]
+                }
+
+            }
+        ]
+    }
+}))
+
+export const JOIN_US_BLADE = {
+    div:{
+        class: "join-us-wrap",
+        children: [
+            {
+                h2: {
+                    content: "Join our community"
+                },
+            },
+            {
+                p: {
+                    content: "Whether you're a part of a DevOps team, an IT engineer, a hobbyist, or a maker, we welcome you to join us in driving forward our vision of a secure, decentralized, and containerized edge."
+                }
+            },
+            {
+                div: {
+                    class: "join-us-wrap-cards",
+                    children: JOIN_US_CARDS,
+                }
+            }
+        ]
+    }
+}
+
+export const BASICS_BLADE = {
+    div: {
+        class: "basics",
+        children: [
+            {
+                title: "Installation",
+                imgSrc: "/index/basics/opened-box.png",
+                text: "See how to get up and running with Kairos, in less than 15 minutes!",
+                href: "https://kairos.io/docs/installation/"
+            },
+            {
+                title: "Architecture",
+                imgSrc: "/index/basics/shell.png",
+                text: "Get inside Kairos, from the factory <br/> to P2P mesh capabilities.",
+                href: "https://kairos.io/docs/architecture/"
+            },
+            {
+                title: "Examples",
+                imgSrc: "/index/basics/origami-bird.png",
+                text: "Stretch your wings with best practices of common tasks after installing Kairos.",
+                href: "https://github.com/kairos-io/kairos/tree/master/examples"
```

Comment:
@mudler should this one point to the repo or to https://kairos.io/docs/examples/ ?